packages/php-wasm/universal/src/lib/universal-php.ts

```diff
@@ -93,6 +93,14 @@ export interface PHPRequest {
 	 * and sent with a `multipart/form-data` header.
 	 */
 	body?: string | Uint8Array | Record<string, string | Uint8Array | File>;
+
+	/**
+	 * Whether to follow redirects.
+	 * - `follow`: Follow redirects.
+	 * - `manual`: Return the redirect response. (Default)
```

Comment:
Should `manual` stay the default behavior or should we `follow` by default like [JS fetch](https://developer.mozilla.org/en-US/docs/Web/API/Request/redirect#value) and browsers do?